doc/intro.rst

```diff
@@ -224,7 +224,9 @@ strings. The following fields are recognized:
 * ``job_status``: string identifying the status of the project. SQUAD
   makes no judgement about its value.
 * ``job_url``: URL pointing to the original test run.
-* ``resubmit_url``: URL that can be used to resubmit the test run.
+* ``resubmit_url``: URL that can be used to resubmit the test run. [As in: this is
+  API-level URL, which, being accessed, would trigged a resubmit (auth is an obvious concern),
+  or this is UI-level page, which user can open, login if needed and press "Resubmit" button?]
```

Comment:
This is actually the URL that was used to resubmit a job. For instance, if you visit a build's testjob tab, you get a list of testjobs and a blue button next to it that allow you to resubmit that job. Then the resubmit-url is that new url, mostly with a new job id in it.